service/vtjclient/build.gradle.kts

```diff
@@ -30,7 +30,7 @@ dependencies {
 }
 
 project.wsdl2javaExt {
-    cxfVersion = "3.3.7"
+    cxfVersion = "3.4.4"
```

Comment:
Doesn't this mean the WSDL stuff should be re-built/-compiled/-whatever-it-is? I just remember trying to update this at some point and everything going wonky with those classes. Maybe I'm misremembering...

Reply:
Yeah this has no practical effect other than removing cxf from the list of "potential updates" in gradle versions plugin, and fixing a CVE.

WSDL generation is indeed broken at the moment and the wsdl2java plugin we use for it has been abandoned, but I've got a fix for this and will submit it in a follow-up pull request